### PR TITLE
Grammar correction in strings.xml

### LINKED ISSUE #687: Grammatical errors in strings.xml
**Description**
There are some grammatical mistakes in the [strings.xml](https://github.com/amahi/android/blob/beta/src/main/res/values/strings.xml) file like missing punctuation marks or inconsistent capitalization. The correction of these mistakes will lead to the enhancement of UX. Also, it will keep in check any other mistakes of such kind that can occur in further translations (for localization) of this [strings.xml](https://github.com/amahi/android/blob/beta/src/main/res/values/strings.xml) file.

**To Reproduce or Feature details**
See any string in the app.

**Expected behavior**
Enhanced user experience and save facing problems ahead by checking mistakes in the  [main string.xml](https://github.com/amahi/android/blob/beta/src/main/res/values/strings.xml) file.

**Smartphone (please complete as much of the following information as possible):**
All devices.

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
@chirag-jn @cpg Kindly review my Pull Request and merge.

### CODE REVIEW

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -324,13 +324,13 @@
 
 <!--    File Options Sheet Dialog-->
     <string name="file_info">File info</string>
-    <string name="sort_by">Sort By</string>
+    <string name="sort_by">Sort bfy</string>
```
Comment (cpg) [inline src/main/res/values/strings.xml]: typo. please correct.

Comment (prakharagarwal-dev) [reply src/main/res/values/strings.xml]: Okay on it.

Comment (prakharagarwal-dev) [reply src/main/res/values/strings.xml]: @cpg corrected the typo.

**Comment by Prakhar-Agarwal-byte** [comment]
@cpg I have corrected the typo. Please merge

**Comment by chirag-jn** [comment]
I have left some comments. Please check the same. Also shouldn't it be `de` instead of `gem` in values folder name and string variables?
Ref: https://stackoverflow.com/questions/7973023/what-is-the-list-of-supported-languages-locales-on-android

File: src/main/res/values/strings.xml [hunk 2]
```diff
@@ -216,20 +216,20 @@
...
-    <string name="preference_title_upload_data">Allow on cellular data</string>
+    <string name="preference_title_upload_server">Select Server</string>
+    <string name="preference_title_upload_share">Select Share</string>
+    <string name="preference_title_upload_location">Upload Location</string>
+    <string name="preference_title_upload_data">Allow on Cellular Data</string>
 
-    <string name="preference_summary_server">Select Server</string>
-    <string name="preference_summary_share">Select Share</string>
-    <string name="preference_summary_location">Select Location</string>
+    <string name="preference_summary_server">Select server</string>
```
Comment (chirag-jn) [inline src/main/res/values/strings.xml:230]: Please follow either sentence case for all sentences or capitalization of the first letter of each word but please don't use both strategies.

**Review by chirag-jn — CHANGES_REQUESTED** [review]

File: src/main/res/values/strings.xml (hunk 2 above)
Comment (prakharagarwal-dev) [reply src/main/res/values/strings.xml:230]: Yes, you are correct, I missed it.
Actually what I was trying to do was to capitalize the first letter of each word which is a title except prepositions and use sentence case for all sentences.
Please tell me what to which one to proceed with.

Comment (prakharagarwal-dev) [reply src/main/res/values/strings.xml:230]: I have made some changes according to your review. Please review them.

**Pull request merged**